tango/local_workspace.py

```diff
@@ -232,6 +233,8 @@ def __init__(self, dir: PathOrStr):
         self.runs_dir = self.dir / "runs"
         self.runs_dir.mkdir(parents=True, exist_ok=True)
         self.step_info_file = self.dir / "stepinfo.sqlite"
+        self.latest_dir = self.dir / "latest"
+        self.latest_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
Why don't you just make `latest/` a link to the most recent run directory? Then we don't have to change anything in case the contents of the run directory change.

Reply:
Yes, I made that simplification as you were reviewing, I think.

Reply:
Sorry, trying to be a bit quicker on the draw today 😅